### DISCUSSION TITLE
Known Rasterio Issue

### DISCUSSION THREAD

**Discussion by pmallas** [anitagraser/movingpandas#224] [General]
I noticed this in your known issues:
On Windows, importing rasterio can lead to DLL errors. If this happens, downgrade the rasterio version to 1.0.13.

I was having a similar problem and I found that by import gdal before rasterio, this went away. GDAL is required by raterio anyway, but seems help rasterio find the correct DLLs.

Just a suggestion to try in case you need newer version of rasterio.

**Comment by anitagraser** [comment]
Thank you for the tipp!
